### ISSUE TITLE
Pressure plate causes red log entry

### ISSUE THREAD

**Issue by crolloS72** [MSchmoecker/Valheim-Pressure-Plate#4]
While building preasure plate caused the following log entry. I use planbuild für building. also the water texture changed.
Just to let you know. I also send you the logoutput.log for your use. 
![image](https://github.com/MSchmoecker/Valheim-Pressure-Plate/assets/74319566/edd12423-2967-41c5-b2fe-5bc4aa8e167e)
![image](https://github.com/MSchmoecker/Valheim-Pressure-Plate/assets/74319566/bd307a12-3970-4d58-9ef5-bba5ce7ccec4)
[LogOutput.log](https://github.com/user-attachments/files/16174547/LogOutput.log)

**Comment by MSchmoecker** [comment]
Thanks for the report, I was able to reproduce the issue. It happens when you place a pressure plate with PlanBuild without integral support, add the required materials and try to open the UI. The plate gets instantly destroyed but is still accessed, causing the NullReferenceException. This will be fixed in the next version.

The water, however, is likely related to other issue in your mod list. The following error is what you have to track down, which is unlikely to be related with PressurePlate:
```
[Error  : Unity Log] ArgumentException: An item with the same key has already been added. Key: 1130726949
Stack trace:
System.Collections.Generic.Dictionary`2[TKey,TValue].TryInsert (TKey key, TValue value, System.Collections.Generic.InsertionBehavior behavior) (at <17d9ce77f27a4bd2afb5ba32c9bea976>:0)
System.Collections.Generic.Dictionary`2[TKey,TValue].Add (TKey key, TValue value) (at <17d9ce77f27a4bd2afb5ba32c9bea976>:0)
ZNetView.Register[T] (System.String name, System.Action`2[T1,T2] f) (at <79989041713a40fba2973f1624473248>:0)
(wrapper dynamic-method) WearNTear.DMD<WearNTear::Awake>(WearNTear)
UnityEngine.Object:Instantiate(GameObject, Vector3, Quaternion)
ZNetScene:DMD<ZNetScene::CreateObject>(ZNetScene, ZDO)
ZNetScene:CreateObjectsSorted(List`1, Int32, Int32&)
ZNetScene:CreateObjects(List`1, List`1)
ZNetScene:CreateDestroyObjects()
ZNetScene:Update()
```

**Comment by MSchmoecker** [comment]
Fixed in 0.9.3
